[build.webkit.org] Changes for running latest buildbot version

aj062 opened the pull request:
#### 6778b6fc6f9c1ae3a42af573785e6c5481943c8e
<pre>
[build.webkit.org] Changes for running latest buildbot version
<a href="https://rdar.apple.com/160642087">rdar://160642087</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=298899">https://bugs.webkit.org/show_bug.cgi?id=298899</a>

Reviewed by Brianna Fan and Jonathan Bedard.

Remove NewStyle suffix to upgrade to latest buildbot.
Buildbot 3.2 removes the &lt;StepType&gt;NewStyle aliases.

* Tools/CISupport/build-webkit-org/steps.py: Remove NewStyle suffix.
* Tools/CISupport/build-webkit-org/master.cfg: Update default text in UI for builds bubbles.
* Tools/CISupport/ews-build/factories.py: Disable running build-webkit-org unit-tests in EWS for now.

Canonical link: <a href="https://commits.webkit.org/300152@main">https://commits.webkit.org/300152@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f6eb455dd6cb7fa4a69f19893d018e40cde9a8c3

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/121391 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/41088 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/31747 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/127832 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/73474 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/9a6ccb8c-0b69-4f77-b622-a33395f34a34) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/123267 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/41790 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/49667 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92209 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61355 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/97304bef-975e-4a27-876a-12a595b917bc) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/124343 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/33367 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/108767 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/72885 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/5727004b-2d00-4cfb-b0b2-1e0967e94260) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/32385 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/26912 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/71412 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/102866 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27086 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/130665 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/48319 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/36738 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/100803 "Passed tests") | | 
| [❌ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/120811 "Failed build.webkit.org unit tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/48687 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/104971 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/100708 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46122 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24188 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/45015 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19259 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/48177 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/47649 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/50995 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/49331 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

briannafan commented on Tools/CISupport/ews-build/factories.py:
```diff
@@ -321,7 +321,8 @@ class ServicesFactory(Factory):
         Factory.__init__(self, platform=platform, configuration=configuration, architectures=architectures, buildOnly=False, additionalArguments=additionalArguments, checkRelevance=True)
         self.addStep(ValidateChange(verifyBugClosed=False, addURLs=False))
-        self.addStep(RunBuildWebKitOrgUnitTests())
+        # TODO: update unit-tests for Buildbot 4 and re-enable RunBuildWebKitOrgUnitTests
```
nit: Can we add a bug link here to track?

aj062 replied:
Sure! Will do.

aj062 committed:
[build.webkit.org] Changes for running latest buildbot version
rdar://160642087
https://bugs.webkit.org/show_bug.cgi?id=298899

Reviewed by Brianna Fan and Jonathan Bedard.

Remove NewStyle suffix to upgrade to latest buildbot.
Buildbot 3.2 removes the <StepType>NewStyle aliases.

* Tools/CISupport/build-webkit-org/steps.py: Remove NewStyle suffix.
* Tools/CISupport/build-webkit-org/master.cfg: Update default text in UI for builds bubbles.
* Tools/CISupport/ews-build/factories.py: Disable running build-webkit-org unit-tests in EWS for now.

Canonical link: https://commits.webkit.org/300152@main
